Kitodo/src/main/java/org/kitodo/production/services/validation/MetadataValidationService.java

```diff
@@ -228,7 +228,7 @@ private ValidationResult checkTheIdentifier(Workpiece workpiece) throws DAOExcep
         String metadataLanguage = user != null ? user.getMetadataLanguage()
                 : Helper.getRequestParameter("Accept-Language");
         return LanguageRange
-                .parse(Objects.nonNull(metadataLanguage) && !metadataLanguage.isEmpty() ? metadataLanguage : "en");
+                .parse(StringUtils.isBlank(metadataLanguage) ? metadataLanguage : "en");
```

Comment:
Should `StringUtils.isBlank" call not inverted or the values switched?

Reply:
`StringUtils` isn't even imported, is it? :anguished:
`MetadataValidationService.java:[231,36] cannot find symbol`

Reply:
No idea. That was a suggestion from @Kathrin-Huber. I only accepted that on GitHub. Apparently, GitHub doesn't check the class for compile errors.